CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 ## 1.4.0 (04-28-2017)
-* [Breaking change] [getThreadHistory](/DOCS.md#getThreadHistory): update parameters
-* [New] [setMessageReaction](/DOCS.md#setMessageReaction)
+* [Breaking change] [getThreadHistory](/DOCS.md#getThreadHistory): update parameters - no more start & end params; replaced with amount
```

Comment:
My reason for that was because start & end was replaced by amount. The [commit](https://github.com/Schmavery/facebook-chat-api/commit/ad25b6b60a252c1b8c3f39780cfe369bde7bb495) shows that timestamp was already part of the function, so it's not a new parameter